cartridge-cli.lua

```diff
@@ -767,21 +776,20 @@ local function detect_name_release_version(source_dir, raw_name, raw_version)
     end
 
     if raw_version then
-        if not normalize_version(raw_version) then
-            die("Passed version '%s' should be semantic (major.minor.patch)",
+        version, release = normalize_version(raw_version)
+        if version == nil then
+            die("Passed version '%s' should be semantic (major.minor.patch[-commit])",
```

Comment:
Maybe `"Passed version '%s' should be semantic (major.minor.patch[-count][-commit])"`?

Reply:
Fixed